### PR TITLE
Fix Interactive_SSHCert_Async test

### PR DESCRIPTION
**Pull request by trwalke**
fix for https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2004

### CODE REVIEW

**Review by pmaytak — APPROVED** [review]
👍

File: tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/SSHCertificateTests.cs [hunk 1]
```diff
@@ -21,8 +21,12 @@ namespace Microsoft.Identity.Test.Integration.SeleniumTests
 {
     public partial class InteractiveFlowTests
     {
+        //This client id is for Azure CLI which is one of the only 2 clients that have PreAuth to use ssh cert feature
+        string _SSH_ClientId = "04b07795-8ddb-461a-bbee-02f9e1bf7b46";
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/SSHCertificateTests.cs:25]: private const s_ssh_clientId

File: tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/SSHCertificateTests.cs [hunk 2]
```diff
@@ -21,8 +21,12 @@ namespace Microsoft.Identity.Test.Integration.SeleniumTests
...
+        //SSH User impersonation scope required for this test
+        private string[] _SSH_scopes = new[] { "https://pas.windows.net/CheckMyAccess/Linux/user_impersonation" };
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/SSHCertificateTests.cs:27]: readonly

Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/SSHCertificateTests.cs:27]: readonly

**Review by bgavrilMS — APPROVED** [review]

**Pull request merged**